include/Reader/readerir.h

```diff
@@ -436,7 +436,7 @@ class GenIR : public ReaderBase {
   };
 
   void rethrow() override { throw NotYetImplementedException("rethrow"); };
-  void returnOpcode(IRNode *Opr, bool SynchronousMethod) override;
+  void returnOpcode(IRNode *Opr, bool SynchronizedMethod) override;
```

Comment:
Nit: Can we make the 2nd param name here consistent with the base class name (since that one will be doxygen'd someday).